src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/components/MethodologyAccordion.tsx

```diff
@@ -0,0 +1,71 @@
+import EditableAccordion from '@admin/components/EditableAccordion';
+import { MethodologyContent } from '@admin/services/methodology/types';
+import { EditingContext } from '@common/modules/find-statistics/util/wrapEditableComponent';
+import orderBy from 'lodash/orderBy';
+import React, { useCallback, useContext } from 'react';
+import useMethodologyActions from '../context/useMethodologyActions';
+import MethodologyAccordionSection from './MethodologyAccordionSection';
+
+export interface MethodologyAccordionProps {
+  isAnnex?: boolean;
+  methodology: MethodologyContent;
+}
+
+const MethodologyAccordion = ({
+  isAnnex = false,
+  methodology,
+}: MethodologyAccordionProps) => {
+  const { isEditing } = useContext(EditingContext);
+  const {
+    addContentSection,
+    updateContentSectionsOrder,
+  } = useMethodologyActions();
+  const sectionKey = isAnnex ? 'annexes' : 'content';
```

Comment:
Is `isAnnex` a bit too restrictive? If we ever want to have other content blocks outside of `annexes` or `content` this will involve changing everything again.

Perhaps we should just pass a`sectionKey` prop through. It also simplifies the methodology actions which end up converting it to a `ContentSectionKey` anyway.